app/views/layouts/_main_nav.html.erb

```diff
@@ -24,10 +24,10 @@
 </nav>
 
 <% if other_nav_links.any? %>
-  <nav class="mb-6 <%= "mt-4" if user_signed_in? %>" aria-labelledby="other-nav-heading">
-    <h2 id="other-nav-heading" class="mt-6 mb-2 fs-base">
+  <nav class="mb-4" aria-labelledby="other-nav-heading">
+    <h3 id="other-nav-heading" class="crayons-subtitle-3 py-2 pl-3">
```

Comment:
I think semantically this makes more sense as an `h2` (since `h3` indicates it has some parent heading, which it doesn't). If we changed this for font size, can we just use a utility class to set this instead? Heading levels are pretty important for assistive technologies

Reply:
fixed.